core/frontend/src/IModelApp.ts

```diff
@@ -445,12 +445,14 @@ export class IModelApp {
     });
 
     // process async onInitialized methods
+    const initPromises: Promise<void>[] = [];

```

Comment:
Why not make this a one-liner: `await this.quantityFormatter.onInitialized()`?

Reply:
I was just trying to provide a location for this and any future async initializations.... but I can change to it to a single line if you prefer.

Reply:
No reason to make it ugly and convoluted until we have more than one thing to initialize.

Reply:
OK, I de-uglified the change.